programs/merkle-distributor/src/state/merkle_distributor.rs

```diff
@@ -58,12 +71,66 @@ pub struct AirdropBonus {
     /// total bonus
     pub total_bonus: u64,
     // vesting slot duration
-    pub vesting_slot_duration: u64,
+    pub vesting_duration: u64,
     /// total bonus
     pub total_claimed_bonus: u64,
 }
 
+pub struct ActivationHandler {
+    /// current slot or current timestamp
+    pub curr_time: u64,
+    /// activation slot or activation timestamp
+    pub activation_time: u64,
```

Comment:
Maybe we update all `activation_slot` / `activation_time` interchangeable variable name to `activation_period` which the name itself is not bounded to time/slot?

Reply:
nice idea, pushed the fix for that